.tekton/central-db-build.yaml

```diff
@@ -10,8 +10,14 @@ metadata:
     pipelinesascode.tekton.dev/max-keep-runs: "500"
     # TODO(ROX-21073): re-enable for all PR branches
     pipelinesascode.tekton.dev/on-cel-expression: |
-      (event == "push" && target_branch.matches("^(master|release-.*)$")) ||
-      (event == "pull_request" && (source_branch.matches("(konflux|renovate|appstudio|rhtap)") || body.pull_request.labels.exists(l, l.name == "konflux-build")))
+      target_branch == "master" && (
+        event == "push" || (
+          event == "pull_request" && (
+            source_branch.matches("(konflux|renovate|appstudio|rhtap)") ||
+            body.pull_request.labels.exists(l, l.name == "konflux-build")
+          )
```

Comment:
Same as https://github.com/stackrox/collector/pull/1957#issuecomment-2472975503

Please allow PR runs for PRs that don't target `master` branch.

Reply:
this should be done now.  (note the latest commit title is inaccurate, but I didn't want to force push an update. I'll reword in the squash message.)